app/presenters/ratings_presenter.rb

```diff
@@ -0,0 +1,4 @@
+class RatingsPresenter < Jsonite
```

Comment:
The presented type name is conventionally singular here. We strayed for some of the other presenters; I'm planning to fix that. Could we fix this one here?